dealership/contracts.py

```diff
@@ -1,12 +1,55 @@
 class Contract(object):
-    pass
-
+    def __init__(self, vehicle, customer):
+        self.vehicle = vehicle
+        self.customer = customer
+        self.discount = None
+        
+        if customer.is_employee():
+            self.discount = .90
+        else:
+            self.discount = 1
+            
 
 class BuyContract(Contract):
     def __init__(self, vehicle, customer, monthly_payments):
-        pass
+        super(BuyContract, self).__init__(vehicle, customer)
```

Comment:
Just out of curiosity, what is the advantage of using the super() function versus using the `__init__()` method to initiate the parent class?